### ISSUE TITLE
required packages may not get installed

### ISSUE THREAD

**Issue by stub42** [juju-solutions/charms.reactive#82]
python3-pip only ~suggests~ _recommends_ build-essentials, python3-wheel and python3-setuptools. If a reactive charm is deployed into a container that already has python3-pip installed, but not the ~suggestions~ _recommends_ packages, then the other packages do not get installed and wheelhouse setup can fail. Which just happened to me, deploying a reactive subordinate to an existing charm.

**Comment by stub42** [comment]
Also, it seems some sites get suggested installs turned off by default causing charms that work elsewhere to fail.

**Comment by stub42** [comment]
To be clear, this means you cannot install a reactive subordinate to a non-reactive primary charm.

The current work around is to list these packages explicitly in layer.yaml, forcing the base layer to install them.

```yaml
options:
    basic:
        packages:
            - build-essential
            - python3-setuptools
            - python3-wheel
            - python3-yaml
```

**Comment by stub42** [comment]
https://github.com/juju-solutions/layer-basic/pull/121

**Comment by johnsca** [comment]
Closed with juju-solutions/layer-basic#121
